ddsenabler_docs/docs/notes/notes.rst

```diff
@@ -5,11 +5,71 @@
 .. include:: forthcoming_version.rst
 
 ##############
-Version v0.1.0
+Version v1.0.0
 ##############
 
+This release includes the following **features**:
+
+* Publish and subscribe to DDS topics in real time.
+* Auto-discovery and management of DomainParticipants, Topics, Types, Publishers and Subscribers via YAML configuration.
+* Flexible JSON serialization and deserialization utilities for DDS samples to enable REST and external integration.
+* Topic allow-listing/deny-listing, QoS tuning, and participant filtering via configuration.
+
+This release includes the following **improvements and bugfixes**:
+
+* Fixed handling of empty configuration file paths and improved log callback handling.
+* Improved robustness of type tests and blackbox test coverage for DDS sample types.
+* Refactored core API for better modularity and clarity.
+
+This release includes the following **CI improvements**:
+
+* Remove deprecated ``Windows-2019`` runner.
+* Use ``tsan`` binaries for TSAN workflows.
+* Add workflow dispatch support to nightly CI.
+* Use Fast DDS master branch and artifacts in workflows.
+* General CI workflow improvements and refactors.
+
+This release includes the following **Documentation changes**:
+
+* Added full documentation set including usage, architecture, and configuration.
+* Included installation instructions for both Linux and Windows (from sources).
+* Added support for building documentation via a CMake option.
+* Fixed broken hyperlinks in documentation.
```

Comment:
Remove these two and add a new item just stating that the documentation includes the API reference.